### ISSUE TITLE
Rename?

### ISSUE THREAD

**Issue by kentcdodds** [kiliman/remix-params-helper#29]
Just a thought. There's nothing remix-specific in this package. This should probably be "zod-params-helper".

**Comment by kiliman** [comment]
That's true. When I first created it, I wasn't sure if I would include Remix-specific stuff in it.

I've actually cleaned up the API for use in my current app, but I wasn't ready to publish since it included some breaking changes. Creating a new package may be the way to go.

Thanks for the nudge.

**Comment by kentcdodds** [comment]
Another note, this library is more than just params. It's also forms which feels different. I wonder if this should be split into two? Just a thought. Either that or the name should reflect what it really does.

**Comment by kiliman** [comment]
I'm actually just renaming it to `zod-helper`.

Its main purpose is to not only validate your input but parse and convert it. Similar to the new `coerce` feature in `zod`.

For example, if the schema is `z.object({ id: z.number() })`, it will verify that the id is a number and convert it automatically. You won't have to use `z.coerce.number()`.

**Comment by kiliman** [comment]
The forms/input helper uses the same schema as the parser/validation, that's why I had it in the same package.

I'll think about the best way to organize it. Thanks!

**Comment by solomonhawk** [comment]
Bumping this because I want to derive input props from (potentially complex) zod schema defs outside of remix and it feels weird to pull in a package called `remix-params-helper` 😄

**Comment by kiliman** [comment]
@solomonhawk Ha, yeah. I built this mainly to help with remix projects, but I realize that it goes beyond that. Thanks for giving me the push. I'll try to clean this up this week.
